test/extended/olm/olmv1.go

```diff
@@ -419,3 +458,47 @@ func checkFeatureCapability(ctx context.Context, oc *exutil.CLI) {
 		g.Skip("Test only runs with OperatorLifecycleManagerV1 capability")
 	}
 }
+
+// verifyAPIEndpoint tests that an API endpoint is accessible and returns valid JSON
+func verifyAPIEndpoint(ctx g.SpecContext, urlPath string) {
+	var lastError error
+
+	// Retry API call a few times to handle potential connection issues
+	err := wait.PollUntilContextTimeout(ctx, 2*time.Second, 30*time.Second, true,
+		func(ctx context.Context) (bool, error) {
+			// Create a custom HTTP client that skips TLS verification
+			tr := &http.Transport{
+				TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
```

Comment:
We should not skip TLS verification. We expect real clients to connect using a trusted CA chain, so our test should use that same CAs as we expect our clients to use.

Reply:
Using a job and curling the http endpoint directly now 👍🏽